src/lib/Microsoft.Health.Events/EventCheckpointing/StorageCheckpointClient.cs

```diff
@@ -152,7 +152,7 @@ public async Task<Checkpoint> GetCheckpointForPartitionAsync(string partitionIde
             }
         }
 
-        public async Task SetCheckpointAsync(IEventMessage eventArgs, IEnumerable<KeyValuePair<Metric, double>> metrics = null)
+        public async Task SetCheckpointAsync(IEventMessage eventArgs, CancellationToken ct, IEnumerable<KeyValuePair<Metric, double>> metrics = null)
```

Comment:
Shouldn't this cancellation token get passed into the call to `UpdateCheckpointAsync`? Where is it being used?

Reply:
yeah, thanks Rob. I updated it